src/button/component.jsx

```diff
@@ -532,6 +535,16 @@ export const Button : Component<ButtonOptions> = create({
                     }
                 }
 
+                if (getDomainSetting('disable_itau')) {
```

Comment:
We should remove this for now -- unless there's a very good reason to add an itau blacklist.